salt/_roster/kubernetes_nodes_roster.py

```diff
@@ -0,0 +1,52 @@
+import fnmatch
+import logging
+log = logging.getLogger(__name__)
+
+try:
+    import kubernetes.client
+    import kubernetes.config
+    HAS_DEPS = True
+except ImportError:
+    log.exception('Unable to load dependencies')
```

Comment:
This log message might not be easily found, a good practice is to return a tuple `(False, "custom error message")` from `__virtual__` (see [here](https://docs.saltstack.com/en/latest/ref/modules/#returning-error-information-from-virtual))

Reply:
Done.